docs/config-options-listings.mdx

```diff
@@ -0,0 +1,38 @@
+---
+id: config-options-listings
+title: Listings For All Buildbuddy Options
+sidebar_label: Options Listings
+---
+
+import FreeServerConfig from '../bazel-bin/server/cmd/buildbuddy/yaml_doc/buildbuddy_server_documented_defaults.mdx'
+import EnterpriseServerConfig from '../bazel-bin/enterprise/server/cmd/server/yaml_doc/buildbuddy_enterprise_server_documented_defaults.mdx'
+import ExecutorConfig from '../bazel-bin/enterprise/server/cmd/executor/yaml_doc/buildbuddy_executor_documented_defaults.mdx'
+
+Provided below are working, documented YAML configs for each Buildbuddy binary
+containing every option that that binary accepts, each set to the default value
+for that option. Any option can be specified on the command line by joining the
```

Comment:
nit: I found the second sentence a bit hard to parse -- maybe split it up and reword slightly like:

Any options that can be specified in the YAML config can also be passed at the command line. For nested options, be sure to write out the full YAML path, with a `.` separating each part.